tests/integration/cattletest/core/test_dynamic_schema.py

```diff
@@ -1,30 +1,57 @@
 from common_fixtures import *  # NOQA
+from gdapi import ApiError
 
 DEFINITION = '''
     {
         "id": "lala",
         "resourceFields": {
              "fooBar": {
-                  "type": "string"
+                  "type": "string",
+                  "description": "foo animal"
              }
         }
     }
     '''
 
 
-def test_schema_create(context, client, super_client):
-    lala = client.by_id_schema('lala')
-    assert lala is None
+def test_schema_lifecycle(context, client, super_client):
+    schema = random_str()
+    gotSchema = client.by_id_schema(schema)
+    assert gotSchema is None
 
-    super_client.create_dynamic_schema(accountId=context.project.id,
-                                       name='lala',
-                                       parent='service',
-                                       definition=DEFINITION)
+    madeSchema = super_client.create_dynamic_schema(
```

Comment:
dont user super client. use a service client, since that is what will actually be used. test_authoration has a service_client function that creates one. You can copy or import that.